### ISSUE TITLE
Calling walk on a DeclarationList with visit: Declaration doesn't work

### ISSUE THREAD

**Issue by fstanis** [csstree/csstree#114]
I'm not sure if this is a bug or I'm doing something wrong, but this doesn't seem to work:

```js
const csstree = require('css-tree');
const ast = csstree.parse('color: red; background-color: blue', { context: 'declarationList' });
csstree.walk(ast, {
  visit: 'Declaration',
  enter: function(node) {
    console.log(node);
  }
});
```

`csstree.walk(ast, function(node) { console.log(node) })` however works fine; also, `visit: 'Identifier'` and anything other than `visit: 'Declaration'` seems to also work as expected.

Any thoughts?

**Comment by lahmatiy** [comment]
That's a bug. Fix is coming!

**Comment by lahmatiy** [comment]
Fixed in [1.0.0-alpha.39](https://github.com/csstree/csstree/releases/tag/v1.0.0-alpha.39)
